src/test/java/com/meta/chatbridge/message/webhook/whatsapp/ChangeTest.java

```diff
@@ -0,0 +1,20 @@
+/*
+ *
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.meta.chatbridge.message.webhook.whatsapp;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ChangeTest {
+
+  String VALID = """
```

Comment:
Do we need this test as is?

Reply:
another great catch. I started writing tests like this and realized it seemed crazy to test like this. I'm going to get test messages from whatsapp in every scenario possible and use those. There are nearly limitless invalid scenarios so testing like this would be crazy.